setup.py

```diff
@@ -0,0 +1,20 @@
+# beprof setup.py
+
+from distutils.core import setup
+
+setup(
+    name = 'beprof',
+    packages = ['beprof'],
+    version = '1.0',
```

Comment:
Make it version 0.1.0. Read more about versioning of the software, i.e. here: http://semver.org, especially fragments related to 

> "How should I deal with revisions in the 0.y.z initial development phase?"
> "How do I know when to release 1.0.0?"